lib/Coffer/Path.hs

```diff
@@ -196,8 +200,16 @@ data QualifiedPath path = QualifiedPath
   { qpBackendName :: Maybe BackendName
   , qpPath :: path
   }
-  deriving stock (Show, Eq, Functor, Generic)
-  deriving anyclass (FromJSON, ToJSON)
+  deriving stock (Show, Eq, Functor)
+
+instance (Buildable path) => ToJSON (QualifiedPath path) where
+  toJSON qPath = do
+    let value = String $ pretty qPath
+    [aesonQQ|
+      {
+        "path": #{value}
+      }
+    |]
```

Comment:
Why do we wrap the path string in a json object with 1 field? Wouldn't `toJSON = String . pretty`, like you did for `EntryPath`, work?

Reply:
Done